src/id-api.js

```diff
@@ -70,24 +71,19 @@ class IDApi {
 
         _private.data.id_info = idInfo;
       },
-      determineSecKey: function() {
-        return new Signature(_private.data.partner_id, _private.data.api_key).generate_sec_key(_private.data.timestamp);
-      },
       determineSignature: function() {
```

Comment:
there is no reason for this to be a function, just inline the signature creation on line 85